browser/extensions/brave_component_loader.cc

```diff
@@ -67,11 +62,6 @@ void BraveComponentLoader::OnComponentReady(std::string extension_id,
   if (allow_file_access) {
     ExtensionPrefs::Get(profile_)->SetAllowFileAccess(extension_id, true);
   }
-#if BUILDFLAG(ETHEREUM_REMOTE_CLIENT_ENABLED)
-  if (extension_id == kEthereumRemoteClientExtensionId) {
-    ReinstallAsNonComponent(kEthereumRemoteClientExtensionId);
```

Comment:
ReinstallAsNonComponent is never referenced now

Reply:
Fixed in ba8f56012e52b119825ef1b98aae6f793bf065ef